tests/glyph_reuse_test.py

```diff
@@ -24,19 +25,30 @@
     "path_a, path_b, expected_result",
     [
         (
-            "M-1,-1 L 0,1 L 1, -1 z",
             "M-2,-2 L 0,2 L 2, -2 z",
-            ReuseResult(glyph_name="A", transform=Affine2D.identity().scale(2)),
+            "M-1,-1 L 0,1 L 1, -1 z",
+            GlyphInstruction(
+                glyph_name="A", path="path_a", transform=Affine2D.identity().scale(0.5)
+            ),
         ),
         # https://github.com/googlefonts/nanoemoji/issues/313
         (
             "M818.7666015625,133.60003662109375 C818.7666015625,130.28631591796875 816.080322265625,127.5999755859375 812.7666015625,127.5999755859375 C809.4529418945312,127.5999755859375 806.7666015625,130.28631591796875 806.7666015625,133.60003662109375 C806.7666015625,136.9136962890625 809.4529418945312,139.60003662109375 812.7666015625,139.60003662109375 C816.080322265625,139.60003662109375 818.7666015625,136.9136962890625 818.7666015625,133.60003662109375 Z",
             "M1237.5,350 C1237.5,18.629150390625 968.870849609375,-250 637.5,-250 C306.1291198730469,-250 37.5,18.629150390625 37.5,350 C37.5,681.370849609375 306.1291198730469,950 637.5,950 C968.870849609375,950 1237.5,681.370849609375 1237.5,350 Z",
-            None,
+            GlyphInstruction(
+                glyph_name=None, path="path_b", transform=Affine2D.identity()
+            ),
         ),
     ],
 )
 def test_glyph_reuse_cache(path_a, path_b, expected_result):
-    reuse_cache = GlyphReuseCache(_DEFAULT_CONFIG.reuse_tolerance)
-    reuse_cache.add_glyph("A", path_a)
-    assert reuse_cache.try_reuse(path_b) == expected_result
+    if expected_result.path == "path_a":
+        expected_result = expected_result._replace(path=path_a)
+    if expected_result.path == "path_b":
+        expected_result = expected_result._replace(path=path_b)
+
+    reuse_cache = GlyphReuseCache(ReusableParts(_DEFAULT_CONFIG.reuse_tolerance))
+    gi = reuse_cache.for_path(path_a)
+    if gi:
```

Comment:
i believe `for_path` method always returns some `GlyphInstruction`, never `None` even when no reuse was found, so `if gi:` is always True.. or am I missing something else?

Reply:
I think there was a point where it could return none and then I changed it :)